sycl/test-e2e/Matrix/get_coord_int8_matB.cpp

```diff
@@ -9,7 +9,9 @@
 
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
-// XFAIL: cpu, gpu-intel-dg2
+
```

Comment:
I suggest adding this instead: `// REQUIRES-INTEL-DRIVER: lin: 30049`

Reply:
Fixed in 7b630036a17c3f346f7be74077e76f01c3caf9ea